From 351c77f8736fe601600718583ace1ace64055fed Mon Sep 17 00:00:00 2001
From: Mark Huang <huangjc@rock-chips.com>
Date: Fri, 30 Oct 2020 10:28:20 +0800
Subject: [PATCH] ARM: configs: Add rv1126-uvc-spi-nand.config

make ARCH=arm rv1126_defconfig rv1126-uvc-spi-nand.config

Signed-off-by: Mark Huang <huangjc@rock-chips.com>
Change-Id: I0ff024ad52b1a94e44cdccb3716f44a91f01e4ef
---
 arch/arm/configs/rv1126-uvc-spi-nand.config | 13 +++++++++++++
 1 file changed, 13 insertions(+)
 create mode 100644 arch/arm/configs/rv1126-uvc-spi-nand.config

diff --git a/arch/arm/configs/rv1126-uvc-spi-nand.config b/arch/arm/configs/rv1126-uvc-spi-nand.config
new file mode 100644
index 000000000000..a7a2f4ec44fe
--- /dev/null
+++ b/arch/arm/configs/rv1126-uvc-spi-nand.config
@@ -0,0 +1,13 @@
+# CONFIG_BT is not set
+# CONFIG_ETHERNET is not set
+# CONFIG_INPUT_TOUCHSCREEN is not set
+# CONFIG_KERNEL_LZ4 is not set
+CONFIG_KERNEL_XZ=y
+# CONFIG_MMC is not set
+# CONFIG_PHYLIB is not set
+# CONFIG_RFKILL is not set
+# CONFIG_SCSI is not set
+# CONFIG_USB is not set
+CONFIG_USB_DWC3_GADGET=y
+# CONFIG_WIRELESS is not set
+# CONFIG_WLAN is not set
-- 
2.35.3

